app/views/games/index.html.erb

```diff
@@ -1,12 +1,374 @@
 <h1>Games</h1>
+<h2>Upcoming</h2>
 
 <% @games.each do |game| %>
-  <div class="cloud">
-    <h2><%= link_to(game.name, game) %></h2>
-    <ul>
-      <li>Start Time: <%= game.start_time %></li>
-      <li>Required Players: <%= game.required_players %></li>
-      <li>Game Type: <%= game.game %></li>
-    </ul>
+  <div class="upcoming">
+    <%= link_to game, class: "game cloud" do %>
+      <h2>
+        <%= game.name %>
+      </h2>
+      <ul>
+        <% if game.start_time && game.start_time.to_date == Date.today %>
```

Comment:
To clean this view code up a bit you could do something like this:

```erb
<% if game.start_time&.today? %>
```

But maybe even nicer to add a `starts_today?` method on Game:

```ruby
class Game < ActiveRecord::Base
  def starts_today?
    start_time&.today?
  end
end
```

```erb
<% if game.starts_today? %>
```